flaskbb-dev.cfg

```diff
@@ -0,0 +1,285 @@
+# This file has been automatically generated on Thursday, 20. May 2021 at 00:45.
```

Comment:
I'd rather not have this file as you'd have to maintain any config changes here as well. Our config generator could handle this.

Also, for docker related flaskbb.cfg config files I'd like to have them in a dedicated docker folder

Reply:
with this i want to automatize all process for deploy include basic config.
i try to use the generator for this cfg file for docker.